spyder/widgets/tabs.py

```diff
@@ -48,6 +48,12 @@ def __init__(self, parent, ancestor):
         self.setAcceptDrops(True)
         self.setUsesScrollButtons(True)
 
+        # Tab name editor
+        self.rename_tabs = rename_tabs
+        if self.rename_tabs is True:

```

Comment:
`self.rename_tabs is True` this is redundant, we can just use

`if self.rename_tabs:`